tools/picard.wdl

```diff
@@ -1023,7 +1025,7 @@ task create_sequence_dictionary {
             )} \
             ~{if defined(fasta_url) then "--URI " + fasta_url else ""} \
             ~{if defined(species) then "--SPECIES " + species else ""} \
-            > ~{outfile_name} \
+            > ~{outfile_name}
```

Comment:
@a-frantz I'm not sure why the syntax highlighting stops after this line. I assumed the stray `\` was the culprit, but it's still broken. Do you see anything?

Reply:
No I don't see anything. Do you know if GitHub ever switched to the syntax highlighting Clay and Peter wrote? I know that was in the works but don't know the status. It might be within our power to patch a fix though.